src/poprox_concepts/domain/article.py

```diff
@@ -4,6 +4,8 @@
 
 from pydantic import BaseModel, ConfigDict, Field
 
+from poprox_concepts.domain.image import Image
+
```

Comment:
I think there should be a change here to add an `images` field to the `Article` class, which will make the unused import warning go away